core/tabs/utils/power-profile.sh

```diff
@@ -45,7 +45,7 @@ configureAutoCpufreq() {
 
     if command_exists auto-cpufreq; then
         # Check if the system has a battery to determine if it's a laptop
-        if [ -d /sys/class/power_supply/BAT0 ]; then
+        if ls /sys/class/power_supply/BAT* >/dev/null 2>&1; then     # Check if the system has a battery (laptop) by looking for any battery directory
```

Comment:
```suggestion
        if ls /sys/class/power_supply/BAT* >/dev/null 2>&1; then
 ```
this is self explanatory, no need for the comment